rwengine/src/objects/PickupObject.cpp

```diff
@@ -143,6 +131,43 @@ void PickupObject::tick(float dt) {
         }
     }
 
+    int coronaColourId = 0;
+#if 0
+    if (modelinfo->name == "colt45")
+        coronaColourId = 1;
+    else if (modelinfo->name == "uzi")
+        coronaColourId = 2;
+    else if (modelinfo->name == "shotgun")
+        coronaColourId = 3;
+    else if (modelinfo->name == "ak47")
+        coronaColourId = 4;
+    else if (modelinfo->name == "m16")
+        coronaColourId = 5;
+    else if (modelinfo->name == "sniper")
+        coronaColourId = 6;
+    else if (modelinfo->name == "rocket")
+        coronaColourId = 7;
+    else if (modelinfo->name == "flame")
+        coronaColourId = 8;
+    else if (modelinfo->name == "molotov")
+        coronaColourId = 9;
+    else if (modelinfo->name == "grenade")
+        coronaColourId = 10;
+    else if (modelinfo->name == "bodyarmour" || modelinfo->name == "bribe")
+        coronaColourId = 12;
+    else if (modelinfo->name == "info" || modelinfo->name == "killfrenzy")
+        coronaColourId = 13;
+    else if (modelinfo->name == "health" || modelinfo->name == "bonus")
+        coronaColourId = 14;
+#endif
+    float timeInMs = engine->getGameTime();
```

Comment:
Since gameTime is currently in seconds, `timeInMs` should probably be named appropriately so if we start counting in milliseconds in the future we'll know what changes need to be made to the units in this function.